Modules/Scripted/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ endif()
 if(Slicer_BUILD_DICOM_SUPPORT)
   list(APPEND modules
     DICOM
+    DICOM2
```

Comment:
I like the idea of this, but it would be nice to have a more descriptive name, like DICOMDirectReader.

Reply:
Thanks for the suggestion! I have updated it to DICOMDirectReader.